### ISSUE TITLE
Bug Report: 

### ISSUE THREAD

**Issue by 8868boyfriend** [ninjamuffin99/Funkin#960]
i got windows seven and 32 bits and everytime i got a black screen and i can play an i can fix

**Comment by aflacc** [comment]
Can you share the Operating System with us? I might be able to help

**Comment by VMGuy23** [comment]
> Can you share the Operating System with us? I might be able to help
Hes on win7 32bit he said

**Comment by aflacc** [comment]
> > Can you share the Operating System with us? I might be able to help
> 
> Hes on win7 32bit he said @aflacc

okay

**Comment by 8868boyfriend** [comment]
I tried many things I downloaded optimized versions I updated the drivers clean my pc I downloaded game optimizer but nothing ... the haxeflixel logo appears and then it turns black but the game is heard and you can play but as I said it is all black

**Comment by VMGuy23** [comment]
Video recording?

**Comment by 8868boyfriend** [comment]
no, I am not recording, I open it normally without permissions and I tried to open it with administrator permissions, but I am not recording anything
